setup.py

```diff
@@ -2,9 +2,11 @@
 
 from setuptools import setup, find_packages
 
+import edx_oauth2_provider
+
 setup(
     name='edx-oauth2-provider',
-    version='1.0.1',
+    version=edx_oauth2_provider.__version__,
```

Comment:
I like this for debugging, when I need to know the version, but don't want to use `pip freeze`. I forget the exact scenario where this would have helped.